components/menu-item/style.scss

```diff
@@ -7,8 +7,9 @@
 	color: $dark-gray-500;
 
 	.dashicon,
-	.components-menu-items__item-icon {
-		margin-right: 5px;
+	.components-menu-items__item-icon,
+	> span > svg { // Also target plugin icons that can have arbitrary classes.
+		margin-right: 4px;
```

Comment:
Oh, this is covered by our weird RTL transform thing, right? I forget how it works. I think this is okay, but I forget.

Reply:
CC: @yoavf it should, right?

Reply:
Yes, RTLCSS will take care of that in RTL mode.

Reply:
Thank you!